src/shapes/Object/InteractiveObject.ts

```diff
@@ -189,29 +189,36 @@ export class InteractiveFabricObject<
    * @param {boolean} forTouch indicates if we are looking for interaction area with a touch action
    * @return {String|Boolean} corner code (tl, tr, bl, br, etc.), or 0 if nothing is found.
    */
-  _findTargetCorner(pointer: Point, forTouch = false): string {
+  _findTargetCorner(
+    pointer: Point,
+    forTouch = false
+  ): { key: string; control: Control } | undefined {
```

Comment:
this return type changed like 4 times in 2023

Reply:
Sign of bad API. If `name` is re-introduced, the return type could just be `: Control`, which is much better IMO.

Reply:
Or add your personal use case whichever it is if is not secret

Reply:
> But it should also be called findTargetControl

I thought about that as well, I'd honestly prefer renaming but did not want to break the API, although it's internal. But if you're okay with it I think it would make more sense.